core/shared/src/main/scala/io/circe/Decoder.scala

```diff
@@ -1,11 +1,13 @@
 package io.circe
 
+import java.nio.ByteBuffer
+
 import cats.Monad
 import cats.data._
 import cats.std.list._
 import cats.syntax.cartesian._
 import cats.syntax.functor._
-import java.util.UUID
+import java.util.{ Base64, UUID }
```

Comment:
Don't we only have `Base64` in Java 8? If so maybe it makes sense to these instances under `circe-java8`?